examples/performance_profiling.cpp

```diff
@@ -69,7 +69,7 @@
 /// Before you run the program, set your `DNNL_VERBOSE` environment
```

Comment:
there are still a few places where `DNNL_VERBOSE` could be replaced by `ONEDNN_VERBOSE`, probably best to fix them all